_posts/2019/2019-12-17-dotjs-2019-in-paris-from-the-perspective-of-a-frontend-designer.md

```diff
@@ -0,0 +1,71 @@
+Even though the concept of both conferences was the same, it differed in some points: there were many more sponsors, speakers, and participants.
+What's more, the conference was two days long and split into two main topics. 
+Day 1 covered Frontend development while day 2 was about Backend development with a focus on JavaScript.
+With the concept of 18-minute talks one after the other, it was possible to attend all of the 20 talks without feeling exhausted, quite the contrary...
+
+{% image_custom image="/assets/img/pages/blog/images/dotjs-2019-speaker.png" width="100" lightbox %}
+
+## My favorite talks
+
+Even though every talk of each speaker was wonderfully prepared and very professionally held, I have my TOP 3 very clear on which I would like to provide some details...
+
+### #1 Charlie Gerard - Exploring the hidden potential of sound data
+
+[Charlie's](https://twitter.com/devdevcharlie){:target="_blank"} talk was about machine learning by "Using the rich properties of sound to gain insights about an activity or environment".
+She demonstrated her topic in a very special way and showed us how a teachable machine recognizes different sounds, for example coughing, typing, or teeth brushing, and how the collected data of a sound look like.
+I mean, how often do you see someone teeth brushing on the stage to get a certain sound into a microphone in front of hundreds of people?
+That was fun to see and exciting for everyone when the teachable machine finally showed the action "teeth brushing".
+You could feel the tension in the room before the silence was replaced by loud applause.
+Charlie also introduced us to the application's possibilities as well as its limitations.
+
+You can find her presentation slides [here](https://docs.google.com/presentation/d/1eXMvepBOs5CT0krFtokZ6ovgMfPNb7TWAPLLFsBPUzk/edit#slide=id.g5bc3345d74_0_16){:target="_blank"}.
+
+### #2 Chris Heilmann - Develop, Debug, Learn?
+
+[Chris'](https://twitter.com/codepo8){:target="_blank"} talk was an answer to a long-standing question for me.
+As a Frontend Designer I would love to code some more, but every time I tried, I failed right from the start.
+The first challenge always was deciding WHAT language or framework from this vast pool of possibilities I'd like to try or learn.
+Followed by the second challenge to SET UP the environment on my own to even get started.
+And the third challenge, START CODING, was sometimes something I didn't achieve at all, because I was already too frustrated.
+Therefore Chris' talk was like a revelation to me.
+He talked about exactly these problems, even if they were looked at from a different angle.
+
+Chris wants us not to think about ourselves, he wants us to think about everyone who uses our final product.
```

Comment:
```suggestion
Chris told us how important it is to know what exactly happens in the background of our written code, besides "Filling the needs", "Adding quality", and "Creating delight". He wants us to focus again on our end-users and not on our comfort zone by dependent ourselves on third party products to develop faster. He questioned sending more code to the devices than necessary because of over-engineered code. It was a kind of a call to go back to the roots.
```

Reply:
Habe dir bei Mattermost geschrieben... 🤗